RELEASES.md

```diff
@@ -1,5 +1,15 @@
 # Releases
 
+## 19.3.5 (March 14th, 2025)
```

Comment:
And again we are NOT going to publish v19.3.5, so we don't need this

The "nightly" build will get generated from main, but this is nightly builds are not officially supported builds